server/events/models/models.go

```diff
@@ -372,6 +373,16 @@ type ProjectCommandContext struct {
 	PolicySets valid.PolicySets
 }
 
+// ProjectCloneDir creates relative path to clone the repo to. If we are running
+// plans and apply in parallel we want to have a directory per project.
+func (p ProjectCommandContext) ProjectCloneDir() string {
+	if p.ParallelPlanEnabled || p.ParallelApplyEnabled {
```

Comment:
Just curious as to why we're enabling this for only parallel use cases? If a deployment happens to enable parallel use cases while a PR is open will there be some weird edge cases?  

I'm of the opinion we should enable it for everything for consistency but not sure if i'm missing something

Reply:
parallel execution is managed per repo. Atlantis deployments can't configure that I think.

Reply:
I think there's a misunderstanding. 

I'm asking why we need this `if` statement at all. But after thinking about it I guess we should leave it in case to minimize any unintended changes for the users of Atlantis.